.vscode-test.mjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { defineConfig } from '@vscode/test-cli';
+// import { apiClient, tokenPluginId } from './src/terraformCloud';
+// import { server } from './src/test/integration/mocks/server';
+
+const config = defineConfig({
+  version: process.env['VSCODE_VERSION'] ?? 'stable',
+  workspaceFolder: process.env['VSCODE_WORKSPACE_FOLDER'] ?? './test/fixtures',
+  launchArgs: ['--disable-extensions', '--disable-workspace-trust'],
+  files: 'out/test/**/*.test.js',
+  mocha: {
+    ui: 'tdd',
+    color: true,
+    timeout: 100000,
+  },
+});
+
+// how to do this?
```

Comment:
I think one way to achieve this, is introducing a setup file and requiring it in the mocha options above.

_src/test/setup.ts_
```ts
/**
 * Copyright (c) HashiCorp, Inc.
 * SPDX-License-Identifier: MPL-2.0
 */

import { apiClient, tokenPluginId } from '../terraformCloud';
import { server } from './integration/mocks/server';

export async function mochaGlobalSetup() {
  apiClient.eject(tokenPluginId);

  server.listen();
}

export async function mochaGlobalTeardown() {
  server.close();
}
```

This makes use of mochas global setup fixtures.

```js
  mocha: {
    ui: 'tdd',
    color: true,
    timeout: 100000,
    require: './out/test/setup.js'
  }
```